Packs/DeCYFIR/Integrations/decyfir/decyfir.yml

```diff
@@ -0,0 +1,69 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: decyfir
+  version: -1
+configuration:
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue:
+  display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: DeCYFIR API key
+  hidden: false
+  name: decyfir_api_key
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 30 days
+  display: How much time before the first fetch to retrieve incidents
+  hidden: false
+  name: first_fetch
+  options:
+  - 10 days
+  - 15 days
+  - 30 days
+  - 60 days
+  - 90 days
+  required: false
+  type: 15
+- defaultvalue: '50'
```

Comment:
Please add additionalinfo key stating that the limitation is according to the category as we discussed.

Reply:
this point i did not get that

Reply:
We said max_fetch = 1 will result in 6 incidents. This needs to be documented.